docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+* **#NetworkCanisterManagement:** installing and upgrading “system” [[NNS Canisters|canisters]] that belong to the network — for example, upgrading the NNS.
+* **#KYC:** proposals that update KYC information for regulatory purposes — for example, during the initial Genesis distribution of ICP in the form of neurons.
+* **#NodeProviderRewards:** topic for proposals to reward node providers.
+
+### Types
+
+Each proposal is an instance of a specific “proposal type”, which determines what information it contains. For each type of proposal, the NNS maintains a corresponding system function, which it invokes whenever a proposal of that type is adopted. When a proposal is adopted by the NNS, it invokes the corresponding system function by drawing information from the proposal’s content to fill the parameters. 
+
+* **ManageNeuron (#NeuronManagement, Restricted Voting)**: this type of proposal calls a major function on a specified target neuron. Only the followers of the target neuron may vote on these proposals, which effectively provides the followees with control over the target neuron. This can provide a convenient and highly secure means for a team of individuals to manage an important neuron. For example, a neuron might hold a large balance, or belong to an organization of high repute, and be publicized so that many other neurons can follow its vote. In both cases, managing the private key of the principal securely could be problematic. (Either a single copy is held, which is very insecure and provides for a single party to take control, or a group of individuals must divide responsibility — for example, using threshold cryptography, which is complex and time-consuming). To address this using this proposal type, the important neuron can be configured to follow the neurons controlled by individual members of a team. Now they can submit proposals to make the important neuron perform actions, which are adopted if and only if a majority of them vote to adopt. (Submitting such a proposal costs a small fee, to prevent denial-of-service attacks.) Nearly any command on the target neuron can be executed, including commands that change the following rules, allowing the set of team members to be dynamic. Only the final step of dissolving the neuron once its dissolve delay reaches zero cannot be performed using this type of proposal, since this would allow control/“ownership” over the locked balances to be transferred. (The only exception to this rule applies to not-for-profit organizations, which may be allowed to dissolve their neurons without using the initial private key.) To prevent a neuron from falling under the malign control of the principal’s private key by accident, the private key can be destroyed so that the neuron can only be controlled by its followers, although this makes it impossible to subsequently unlock the balance.
```

Comment:
```suggestion
* **ManageNeuron (#NeuronManagement, Restricted Voting)**: this type of proposal calls a function on a specified target neuron. Only the followers of the target neuron may vote on these proposals, which effectively provides the followees with control over the target neuron. This can provide convenient means for a team of individuals to manage an important neuron. For example, a neuron might hold a large stake, or belong to an organization of high repute, and be publicized so that many other neurons can follow its vote. In both cases, managing the private key of the principal securely could be problematic. (Either a single copy is held, which is very insecure and provides for a single party to take control, or a group of individuals must divide responsibility — for example, using threshold cryptography, which is complex and time-consuming). To address this using this proposal type, the important neuron can be configured to follow the neurons controlled by individual members of a team. Now they can submit proposals to make the important neuron perform actions, which are adopted if and only if a majority of them vote to adopt. (Submitting such a proposal costs a small fee, to prevent denial-of-service attacks.) Nearly any command on the target neuron can be executed, including commands that change the following rules, allowing the set of team members to be dynamic. Only the final step of dissolving the neuron once its dissolve delay reaches zero cannot be performed using this type of proposal, since this would allow control/“ownership” over the locked balances to be transferred. (The only exception to this rule applies to not-for-profit organizations, which may be allowed to dissolve their neurons without using the initial private key.) To prevent a neuron from falling under the malign control of the principal’s private key by accident, the private key can be destroyed so that the neuron can only be controlled by its followers, although this makes it impossible to subsequently unlock the balance.
```

I think this is all true but I wonder how relevant this is. If we think this is needed for develeopers I wouuls still put this to the very end. 
I think there is no way to create a neuron that is not-for-profit. Only some neurons at genesis had this field. If we want devs to use this they might need to know this. But I wonder if we should recommend that